web/yo/app/scripts/directives/driverealtimestring.js

```diff
@@ -119,13 +119,23 @@ angular.module('oncokbApp')
                         scope.content.propagation = scope.es.get('propagation');
                     }
                     scope.changePropagation(true);
-                    scope.$watch('content.propagation', function(n) {
+                    scope.$watch('content.propagation', function(n, o) {
                         if (!_.isUndefined(scope.es)) {
                             if (_.isUndefined(scope.es.get('propagation')) ||
                                 scope.es.get('propagation') !== n) {
                                 scope.es.set('propagation', n);
                             }
                         }
+                        scope.rs.set('lastReviewedPropagation', o);
+                        var uuid = scope.uuid.getText();
+                        var tempMapping = $rootScope.reviewMeta.get(uuid);
+                        if(!tempMapping) {
+                            tempMapping = $rootScope.metaModel.createMap();
+                        }
+                        tempMapping.set('review', true);
```

Comment:
Until now, I still don't quite sure about this review property? Does is mean 'to be reviewed', 'reviewed' or something else since it is a Boolean property. We don't have to change for this property this time but in the future, can we avoid this ambiguity naming?